doc/book/admin/monitoring/metrics_reference.rst

```diff
@@ -0,0 +1,1011 @@
+        *   -   ``tnt_synchro_queue_owner``
+            -   Instance ID of the current synchronous replication master.
+
+        *   -   ``tnt_synchro_queue_term``
+            -   Current queue term.
+
+        *   -   ``tnt_synchro_queue_len``
+            -   How many transactions are collecting confirmations now.
+
+        *   -   ``tnt_synchro_queue_busy``
+            -   Whether the queue is processing any system entry (CONFIRM/ROLLBACK/PROMOTE/DEMOTE).
+
+..  _metrics-reference-election:
+
+Election
+--------
+
+Shows the current state of a replica set node in regards to leader election.
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   -   ``tnt_election_state``
+            -   election state (mode) of the node.
+                When election is enabled, the node is writable only in the leader state.
+                Possible values:
+
+                *   0 (``follower``) -- all the non-leader nodes are called followers
+                *   1 (``candidate``) -- the nodes that start a new election round are called candidates.
+                *   2 (``leader``) -- the node that collected a quorum of votes becomes the leader
+
+        *   -   ``tnt_election_vote``
+            -   ID of a node the current node votes for.
+                If the value is 0, it means the node hasn’t voted in the current term yet.
+
+        *   -   ``tnt_election_leader``
+            -   Leader node ID in the current term.
+                If the value is 0, it means the node doesn’t know which node is the leader in the current term.
+
+        *   -   ``tnt_election_term``
+            -   Current election term.
+
+        *   -   ``tnt_election_leader_idle``
+            -   Time in seconds since the last interaction with the known leader.
+
+Memtx
+-----
+
+Memtx mvcc memory statistics.
+Transaction manager consists of two parts:
+- the transactions themselves (TXN section)
+- MVCC
+
+..  _metrics-reference-memtx_txn:
+
+TXN
+~~~
+
+..  container:: table
+
+    ..  list-table::
+        :widths: 25 75
+        :header-rows: 0
+
+        *   - ``tnt_memtx_tnx_statements`` are the transaction statements.
+            -   For example, the user started a transaction and made an action in it `space:replace{0, 1}`.
+                Under the hood, this operation will turn into ``statement`` for the current transaction.
+                This metric always has the label ``{kind="..."}``,
+                which has the following possible values:
+
+                *   ``total``
+                    The number of bytes that are allocated for the statements of all current transactions.
+                *   ``average``
+                    Average bytes used by transactions for statements
+                    (`txn.statements.total` bytes / number of open transactions).
+                *   ``max``
+                    The maximum number of bytes used by one the current transaction for statements.
```

Comment:
```suggestion
                *   ``total`` -- the number of bytes that are allocated for the statements of all current transactions.
                *   ``average`` -- average bytes used by transactions for statements
                    (`txn.statements.total` bytes / number of open transactions).
                *   ``max`` -- the maximum number of bytes used by one the current transaction for statements.
```
I suggest one of the following formatting:

*   ``total`` -- the number of bytes...
*   ``total``:  the number of bytes ...

Also, please check this list formatting in other reference - let's make it consistent

Reply:
Thanks, updated using the second approach (with `:`).